models/BaseConfig.cfc

```diff
@@ -1103,6 +1107,49 @@ component accessors="true" {
 		return this;
 	}
 
+
+	/**
+	* Add a single component Path to the config
+	*
+	* @physical The physical path that the engine should search
+	* @archive Path to the Lucee/Railo archive
+	* @name Name of the Component Path
+	* @inspectTemplate String containing one of "never", "once", "always", "" (inherit)
+	* @primary Strings containing one of "physical", "archive"
+
+
+		inspect-template	
+		physical	
+		primary	
+		readonly	
+		virtual	
+
+
+	*/
+	function addComponentPath(
+			required string name,
+			required string physical,
+			string archive="",
+			string primary="physical",
+			string inspectTemplate="never",
```

Comment:
This default doesn't match the admin UI.  Component mappings default to "inherit", which is stored as an empty string.  FWIW, I don't tend to default anythign here.  If it's no set, it's not set.  Otherwise, this can make the JSON exports contain data that was never really in the XML in the first place.  I leave anything that's not set null so long as it's not required and I only default it at the last minute when writing out the XML.  The same goes for the `primary` `archive` arguments.

Reply:
Good plan. Updated. I am not sure if I need to do anything apart from push my changes?